lib/tasks/foreman_templates_tasks.rake

```diff
@@ -57,6 +57,13 @@ namespace :templates do
     }).purge!
   end
 
+  desc 'Clean data created by this plugin, this will permanently delete the data!'
```

Comment:
Overly dramatic? It implies it'll delete the *templates* which clearly isn't true. Perhaps:
```
Clean up for uninstallation- this removes all custom Settings added by the plugin
```
or similar?